fix: CI docker build

Building Docker image on CI fails with:
```
#52 1353.1 error[E0463]: can't find crate for `core`
#52 1353.1   |
#52 1353.1   = note: the `aarch64-unknown-linux-gnu` target may not be installed
#52 1353.1   = help: consider downloading the target with `rustup target add aarch64-unknown-linux-gnu`
```

--- a/.github/workflows/docker-publish.yaml
+++ b/.github/workflows/docker-publish.yaml
@@ -2,6 +2,7 @@ name: Fendermint CI - Publish
 
 # This workflow is triggered from the main CI workflow.
 on:
+  workflow_dispatch:
   workflow_call:
 
 env:

--- a/fendermint/docker/builder.ci.Dockerfile
+++ b/fendermint/docker/builder.ci.Dockerfile
@@ -72,7 +72,9 @@ RUN \
   set -eux; \
   case "${TARGETARCH}" in \
   amd64) ARCH='x86_64'  ;; \
-  arm64) ARCH='aarch64' ;; \
+  arm64) ARCH='aarch64'; \
+      rustup target add aarch64-unknown-linux-gnu; \
+      rustup toolchain install stable-aarch64-unknown-linux-gnu ;; \
   *) echo >&2 "unsupported architecture: ${TARGETARCH}"; exit 1 ;; \
   esac; \
   rustup show ; \
@@ -94,7 +96,9 @@ RUN \
   set -eux; \
   case "${TARGETARCH}" in \
   amd64) ARCH='x86_64'  ;; \
-  arm64) ARCH='aarch64' ;; \
+  arm64) ARCH='aarch64'; \
+    rustup target add aarch64-unknown-linux-gnu; \
+    rustup toolchain install stable-aarch64-unknown-linux-gnu ;; \
   esac; \
   cargo install --locked --root output --path fendermint/app --target ${ARCH}-unknown-linux-gnu ; \
   cargo install --locked --root output --path ipc/cli --target ${ARCH}-unknown-linux-gnu

